docusaurus/docs/create-a-plugin/extend-a-plugin/fetch-data-from-frontend.md

```diff
@@ -27,46 +27,51 @@ This guide explains how the data proxy works and explores common issues in its u
 
 Instead of performing a request directly from the browser to the server, you perform the request through the Grafana backend server, which handles it and returns the response to the plugin.
 
-- **Without data proxy**: Without a data proxy the requests go directly from the browser to the third-party server.
-- **With data proxy**: Because the request to the third-party server happens from the Grafana backend, there are no restrictions in CORS, and you can instruct Grafana to send the request authenticated or using sensitive data stored in the plugin configuration.
+- **Without data proxy**: The requests go directly from the browser to the third-party server.
+- **With data proxy**: The requests go from the browser to the Grafana backend and then to the third-party server. In this case, there are no restrictions in CORS, and you can instruct Grafana to send the request authenticated or using sensitive data stored in the plugin configuration.
```

Comment:
Nit
```suggestion
- **With data proxy**: The requests go from the browser to the Grafana backend and then to the third-party server. In this case, there are no restrictions in CORS, and you can instruct Grafana to send the request authenticated by using sensitive data stored in the plugin configuration.
```

Reply:
Do we have any examples below of using sensitive data for proxy route headers? If yes it might makes sense to link from here?